projects/isy-angular-widgets/src/lib/input-char/components/multi-select-button/multi-select-button.component.spec.ts

```diff
@@ -0,0 +1,237 @@
+import {MultiSelectButtonComponent} from './multi-select-button.component';
+import {createComponentFactory, Spectator} from '@ngneat/spectator';
+import {MockModule} from 'ng-mocks';
+import {AccordionModule} from 'primeng/accordion';
+import {SelectButton, SelectButtonModule} from 'primeng/selectbutton';
+import {FormsModule} from '@angular/forms';
+import sonderzeichenliste from '../../sonderzeichenliste.json';
+import {InputCharData, Schriftzeichengruppe, Zeichenobjekt} from '../../model/model';
+import {By} from '@angular/platform-browser';
+import {ComponentFixture} from '@angular/core/testing';
+import {InputCharModule} from '@isy-angular-widgets/public-api';
+
+let spectator: Spectator<MultiSelectButtonComponent>;
+let component: MultiSelectButtonComponent;
+let fixture: ComponentFixture<MultiSelectButtonComponent>;
+
+const charset = sonderzeichenliste as Zeichenobjekt[];
+const bases = [...new Set(charset.map((item) => (item.grundzeichen === '' ? '*' : item.grundzeichen)))];
+const groups = [...new Set(charset.map((item) => item.schriftzeichengruppe))];
+
+const headerStr = 'All';
+const inputData: InputCharData = {Base: bases, Groups: groups};
+const props = {
+  dataToDisplay: inputData,
+  allButtonOptionsLabel: headerStr
+};
+
+describe('Unit Tests: MultiSelectButtonComponent', () => {
+  const createComponent = createComponentFactory({
+    component: MultiSelectButtonComponent,
+    imports: [MockModule(AccordionModule), MockModule(SelectButtonModule), MockModule(FormsModule)]
+  });
+
+  beforeEach(() => {
+    spectator = createComponent({props: props});
+    component = spectator.component;
+    fixture = spectator.fixture;
+    spectator.detectChanges();
+  });
+
+  it('should create', () => {
+    expect(component).toBeTruthy();
+  });
```

Comment:
Duplicated Test: is in Unit Test section and Integration Test section (Line 170)

Reply:
Deleted